plugin/geodns/filter.go

```diff
@@ -49,14 +44,6 @@ func NewResponseFilter(w dns.ResponseWriter, filter *filter, client net.IP) *Res
 // WriteMsg records the message and its length written to it and call the
 // underlying ResponseWriter's WriteMsg method.
 func (r *ResponseFilter) WriteMsg(res *dns.Msg) error {
-	qtype := res.Question[0].Qtype
-	qName := res.Question[0].Name
-
-	if !isSupportedType(qtype) {
```

Comment:
Maybe we should keep this. Despite we check type in `(g GeoDNS) ServeDNS(...)` function, the `ResponseFilter` is separated component. Though it's up to you.